worker/migrationmaster/worker.go

```diff
@@ -432,12 +438,66 @@ func (w *Worker) doSUCCESS(status coremigration.MigrationStatus) (coremigration.
 	return coremigration.LOGTRANSFER, nil
 }
 
-func (w *Worker) doLOGTRANSFER() (coremigration.Phase, error) {
-	// TODO(mjs) - To be implemented.
-	// w.setInfoStatus("successful: transferring logs to target controller")
+func (w *Worker) doLOGTRANSFER(targetInfo coremigration.TargetInfo, modelUUID string) (coremigration.Phase, error) {
+	err := w.transferLogs(targetInfo, modelUUID)
+	if err != nil {
+		return coremigration.UNKNOWN, errors.Trace(err)
+	}
 	return coremigration.REAP, nil
 }
 
+func (w *Worker) transferLogs(targetInfo coremigration.TargetInfo, modelUUID string) error {
+	sent := 0
+	reportProgress := func(sent int) {
+		w.setInfoStatus("successful, transferring logs to target controller (%d sent)", sent)
```

Comment:
This will write to the logs - as well as feeding the migration status shown in the "juju status" output. I think the progress reporting is useful but agree that 1000 is probably too frequent. Making it percentage based or logging an update every N seconds might be better.

FWIW the migrationmaster already reports progress every 30s when waiting for minions to report back at various migration phases. Maybe that const should be shared?